autopilot/multiwaypointfollower.cpp

```diff
@@ -60,6 +60,34 @@ void MultiWaypointFollower::resetState()
     mWayPointFollowerList[mWaypointFollowerID]->resetState();
 }
 
+double MultiWaypointFollower::getPurePursuitRadius() const
+{
+    return qSharedPointerCast<PurepursuitWaypointFollower>(mWayPointFollowerList[mWaypointFollowerID])->getPurePursuitRadius();;
+}
+
+void MultiWaypointFollower::setPurePursuitRadius(double value)
+{
+    for (auto wayPointFollower = mWayPointFollowerList.begin(); wayPointFollower != mWayPointFollowerList.end(); ++wayPointFollower)
+        qSharedPointerCast<PurepursuitWaypointFollower>(*wayPointFollower)->setPurePursuitRadius(value);
```

Comment:
Would look nicer this way in my opinion (applies to following loops as well):
```suggestion
    for (const auto &wayPointFollower : mWayPointFollowerList)
        qSharedPointerCast<PurepursuitWaypointFollower>(wayPointFollower)->setPurePursuitRadius(value);
```